package.json

```diff
@@ -6,16 +6,8 @@
     "lint": "turbo run lint",
     "release": "lerna publish --no-private"
   },
-  "workspaces": [
-    "packages/*",
-    "apps/*"
-  ],
```

Comment:
pnpm only respects the `pnpm-workspace.yaml` afaik

Reply:
:o ty